client/extensions/woocommerce/app/dashboard/required-plugins-install-view.js

```diff
@@ -131,12 +137,14 @@ class RequiredPluginsInstallView extends Component {
 					progress,
 				} ) );
 				this.props.installPlugin( site.ID, wporgPlugin );
+				this.setUpdateTimeout();
 				return;
```

Comment:
The API starts getting really funky when you start installing in parallel.  It starts returning odd errors and sometimes only gets halfway through an install and then the api and wp-admin are 500 errors.